src/components/cohortPages/Description.astro

```diff
@@ -15,7 +15,9 @@ const { takeaways } = Astro.props;
       {
         takeaways.map((takeaway) => (
           <li class="my-4 lg:text-xl">
-            <span class="font-header">{takeaway.name}</span>{" "}
+            <span class="font-header font-bold">
+              {takeaway.name}
+            </span>{" "}
```

Comment:
is the `{" "}` needed

Reply:
We currently are in the middle of design change, where one cohort page is redesigned, but others are not. once all the pages are redesigned and have a similar layout, we can remove duplicate components and use one for all